.changeset/modern-mugs-shout.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/core-components': patch
+---
+
+Adds prop to SupportButton to override support config items and adds the option to show custom list items per page
```

Comment:
Had a hard time reading this tbh 😅. Could perhaps rewrite it a bit, highlighting the new `items` prop that can be used to override what would otherwise be grabbed from config?

Reply:
Hopefully it is a bit better now. I agree it was a bit hard to read